### DISCUSSION TITLE
Persistence Plugin: Do You Install The Distro, Or Try The Distro?

### DISCUSSION THREAD

**Discussion by trymeouteh** [ventoy/Ventoy#1365] [Q&A] [answered]
When you setup the Persistence Plugin with a Linux distro, do you install the distro to the persistent storage or do you never install the distro and when you boot up you always click on "Try Ubuntu" instead of "Install Ubuntu" for example.

I realize the files I made in my documents showed up which I created in "Try Ubuntu" mode and when I booted back up again, the files were still there. Or is it still proper to install Ubuntu onto the persistent storage?

**Comment by ventoy** [comment, marked as answer]
Try the distro.
Persistence plugin is just for those who don't want to install the distro to a persistence storage and just want to run the live environment (try mode).
When you select try Ubuntu, then the current ubuntu system is running in the memory, so if you don't use persistence plugin, then all the data you created will be lost once you reboot it because all the data are saved in the memory not in a persistence storage.

Some one don't want to install the distro to a persistence storage and just want to use the live environment (like always try Ubuntu), but they also want to save a little data after boot, then they can use persistence plugin.

If you install Ubuntu to a persistence storage, then there is no need to use  persistence plugin anymore.

**Comment by trymeouteh** [comment]
Instead of persistence storage (memory), is there a way to install a distro on a ventoy usb drive like the Windows VHD plugin were the OS is actually installed and bootable within ventoy boot menu?

I would like to find a way preferably to install a distro onto a USB drive and encrypt the installation with the distro LUKS encryption that most distros come with, therefore having a portable USB drive that can boot up a Linux OS that is encrypted.

**Reply by ventoy** [reply]
https://www.ventoy.net/en/plugin_vtoyboot.html

**Comment by flavi1** [comment]
Documentation says `Just put a persistence data file in the 1st partition and tell Ventoy by the json configuration, and that's all.`
It would be great if we will be able to put the persistence file in another disc/partition.
Possible use case : Live iso is in a SSD, and persistence file in a HDD.
In the distro, a script can sometimes generate a new iso. Then we obtain a less as possible writing on SSD resulting a long life cycle for the SSD... (just a suggestion, not an absolute needed request at all)
